README.md

```diff
@@ -70,8 +70,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- coming soon after aprroved ....
-- coming soon after approved ....
+- coming soon ....
```

Comment:
- [ ] As pointed out by the previous reviewer, please kindly ensure that the site is hosted properly and the link included in the README.